src/nexpy/gui/pyqt.py

```diff
@@ -1,8 +1,10 @@
+import matplotlib
 import sip
 for api in ['QString', 'QVariant']:
     sip.setapi(api, 2)
 
-from qtconsole.qt import QtCore, QtGui, QtSvg, QT_API
+matplotlib.use('Qt4Agg')
```

Comment:
The problem with using qtconsole to choose the PyQt version is that it is likely to return PyQt5, which causes the other issues I mentioned. The Matplotlib imports can be forced to use PyQt4, so is safer until the PyQt5 issues are resolved.